created ide session unknown error page following slack message (#5960)
dbt-labs/docs.getdbt.com: website/docs/faqs/Troubleshooting/ide-session-unknown-error.md

---
title: I'm receiving an 'Your IDE session experienced an unknown error and was terminated. Please contact support'.
description: "Add a repository when seeing IDE unknown error"
sidebar_label: 'Receiving unknown error in the IDE'

> id: IDE

Reviewer: sugg removing since it's unnecessary. what you have as the filename is a good URL ie `https://docs.getdbt.com/faqs/Troubleshooting/ide-session-unknown-error`

(side note: as a best practice, if the `id` key is used, the value should be in all lower case (no upper case) to avoid Vercel issues.)

```suggestion
```

---

> If you're experiencing the below error when you try to open the cloud IDE, it may be due to a few scenarios but it typically indicates a missing repository.

Reviewer: sugg:
- iirc we prefer to use "the following" (vs "below") for accessibility 
- "may" should be used to mean _permission_ like `you may sit down`. "may" has several meanings depending on context so this convention is for translation purposes

```suggestion
If you're seeing the following error when you launch the dbt Cloud IDE, it could be due to a few scenarios but, commonly, it indicates a missing repository:
```

```shell

> Your IDE session experienced an unknown error and was terminated. Please contact support

Reviewer: sugg: missing period

```suggestion
Your IDE session experienced an unknown error and was terminated. Please contact support.
```

```

> To resolve this, we recommend adding a repository like a [managed repository](/docs/collaborate/git/managed-repository) or your preferred Git account by navigating to Project > Repository and selecting your repository.

Reviewer: since we don't know if these steps will actually "resolve this" bc there are several reasons a user could run into this error, sugg slightly diff wording. 

something like this or similar: 

also, for dropdown menus, we bold the UI text

```suggestion
You can try to resolve this by adding a repository like a [managed repository](/docs/collaborate/git/managed-repository) or your preferred Git account. To add your Git account, navigate to **Project** > **Repository** and select your repository.
```

> If you've tried the steps above and are still experiencing this behavior - reach out to the Support team at support@getdbt.com and we'll be happy to help.

Reviewer: sugg: more concise

```suggestion
If you're still running into this error, please contact the Support team at support@getdbt.com for help. 
```
